pkg/injector/patcher/sidecar.go

```diff
@@ -56,8 +56,6 @@ type SidecarConfig struct {
 	RemindersService            string
 	SentrySPIFFEID              string
 	SidecarHTTPPort             int32 `default:"3500"`
```

Comment:
for consistency, do we want to make this one configurable as well?

Reply:
We could, however I'd suggest doing it in a separate PR and this is less pressing than changing 50002 as 3500 is within the non-dynamic range.